starter/host.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "2.0",
+    "logging": {
+        "applicationInsights": {
+            "samplingExcludedTypes": "Request",
+            "samplingSettings": {
+                "isEnabled": true
+            }
+        }
+    },
+    "extensions": {
```

Comment:
I thought this was getting nixed?

Reply:
Point of time... I will nix

Reply:
Oh... you were talking about the prefix - `api` here. No, I am going with your suggestion and keeping the default here, to avoid confusion